easybuild/easyconfigs/a/ASE/ASE-3.15.0-foss-2017b-Python-3.6.2.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.15.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5e22d961b1311ef4ba2d83527f7cc7448abac8cf9bddd1593bee548459263fe8']
+
+dependencies = [
+    ('Python', '3.6.2'),
+    ('matplotlib', '2.1.0', '-Python-%(pyver)s'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ase'],
+    'dirs': [],
```

Comment:
@schiotz Existing `ASE` easyconfigs have a more extensive list of paths here, why did you trim things down to just `bin/ase`?

Reply:
@boegel Because the command line tools have been merged into a single tool (ase), the remaining scripts (ase-gui etc) are deprecated, and are just wrapper scripts for the main ase tool for backward compatibility.  They will soon go away completely.

Reply:
OK, thanks for clarifying!